structures/message.js

```diff
@@ -0,0 +1,24 @@
+const s = require('superstruct');
+const { ZuluDateTimeStruct } = require('./lib');
+const uuid = require('uuid');
+const isUuid = require('is-uuid');
+
+const Uuid = s.define('Uuid', isUuid.v4);
```

Comment:
I'm pretty sure that you need it as un function when called, but test it first.
```suggestion
const Uuid = s.define('Uuid', isUuid.v4());
```